### PR TITLE
CCX-4908: Add Db logging setup

### PR DESCRIPTION
**Pull request by ram-vibhul**

### CODE REVIEW

**Comment by ram-vibhul** [comment]
> Fluentbit part is obsolete now and not configurable? Do we even want users/clients to tamper that much with loki configuration? IMHO it should only be briefly mentioned and detailed in the helm-chart. These docs do not explain what this loki is, why we have it and what it is used for.

Fluentbit is configurable but its already provided as comments in helm-ccx values.yaml. if you want, then i will add here. Just don't want this docs to get bloated

File: docs/admin/Installation/Logging.md [hunk 1]
```diff
@@ -1,172 +1,48 @@
...
-        Name                tail
-        Tag                 ccx.mysql
-        Path                /var/log/mysql/mysqld.log
-        DB                  /var/fluent-bit/state/flb_mysql.db
-        Mem_Buf_Limit       5MB
-        Skip_Long_Lines     On
-        Refresh_Interval    10
-        Read_from_Head      false
+  loki:
+    loki_host_url: &loki_host_url "REPLACE_ME" # Set your custom Loki FQDN here
```
Comment (bunin) [inline docs/admin/Installation/Logging.md:25]: why would I need that? where do I get FQDN?

**Review by bunin — APPROVED** [review]

File: docs/admin/Installation/Logging.md (hunk 1 above)
Comment (ram-vibhul) [reply docs/admin/Installation/Logging.md:32]: it will be automatically set if you didn't specify fqdn. We offer the customer to choose fqdn if they want

Comment (ram-vibhul) [reply docs/admin/Installation/Logging.md:32]: this is needed to forward the logs to loki instance

**Comment by ram-vibhul** [comment]
> to forward logs to their logging system they'd add additional config to fluentbit.

@oskar-s9s Why to split loki and fluentbit? Those are Coupled together to make it work.

There are 2 steps,  one is it works without any configs, second is customers can configure their required fqdn if they don't want to use default ccxfqdn. So you want to remove this second point?
> Given our loki is internal to CCX it should really work w/o any further configuration.

I have already added docs on how to add extraOutputConfig to forward logs.
> And if users want to forward logs to their logging system they'd add additional config to fluentbit.

**Pull request merged**